### DISCUSSION TITLE
Is StreamGeometry Used to Render Material Icons?

### DISCUSSION THREAD

**Discussion by anose001** [AvaloniaCommunity/Material.Avalonia#394] [Q&A]
Does this library use StreamGeometry to render material icons? I was not able to find a lot in the repo except at a few locations.

**Comment by SKProCH** [comment]
Nope. Icons extracted to separate package - [Material.Icons](https://github.com/SKProCH/Material.Icons) to reduce the package size (since no everyone using the icons).

Material.Icons uses Geometry.Parse to parse the svg path string.
